docs/traffic-policy/examples/index.mdx

```diff
@@ -32,14 +32,38 @@ Explore a curated collection of examples and configuration examples spanning fro
 
 A number of these examples come from a longer article about how ngrok [makes policy management accessible](https://ngrok.com/blog-post/api-gateway-policy-management-examples) to developers, including a simple Go-based application for testing these and other configurations.
 
-See the following categories for specific expressions and actions:
+## Route requests
 
-- [Authentication](#authentication)
-- [Rate limiting](#rate-limiting)
-- [Block unwanted requests](#block-unwanted-requests)
-- [Other](#other)
```

Comment:
These links were broken, but I just removed this content entirely because all it does is recreate the TOC. In general I frown upon that because now there are two TOCs to maintain on the page, and if the links are ever broken it looks unprofessional